docs/sxpswap/users/trust-wallet.md

```diff
@@ -4,6 +4,34 @@ title: Swap/Migration for Trust Wallet users
 
 # Introduction
 
-!!! danger annotate "Coming soon"
+!!! danger annotate "Disclaimer!"
 
-    The implementation for using the swap bridge with the Trust Wallet is currently being implemented and will be available in early Q2.
+    Please read the information on this page carefully.
+    You need a Solar Wallet address to perform the swap from ERC20/BEP20 SXP to Mainnet SXP.
+    Visit our official website to download the latest [Solar Desktop Wallet](https://solar.org/desktop-wallet).
```

Comment:
Links like these should be current and open in a new tab when the destination URL is external.

<ins>`https://solar.org/desktop-wallet` => `https://solar.org/wallets`</ins>  

`[Solar Desktop Wallet](https://solar.org/desktop-wallet)` => `<a href="https://solar.org/wallets" target="_blank" rel="noopener noreferrer">Solar Desktop Wallet</a>`